### ISSUE TITLE
Update Manifest

### ISSUE THREAD

**Issue by BLACK4585** [Cloud9Developer/Jellyfin-Newsletter-Plugin#91]
I just notice we can't get the update since it's not availible in the manifest. Maybe this also fixes the following error:
```
System.IO.FileNotFoundException: Could not load file or assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.

File name: 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'
   at Jellyfin.Plugin.Newsletters.Shared.DATA.SQLiteDatabase..ctor()
   at Jellyfin.Plugin.Newsletters.Scanner.NLDataGenerator.NewsletterDataGenerator..ctor()
   at Jellyfin.Plugin.Newsletters.Scripts.SCRAPER.Scraper..ctor(ILibraryManager libraryManager, IProgress`1 passedProgress, CancellationToken cancellationToken)
   at Jellyfin.Plugin.Newsletters.ScheduledTasks.ScanLibraryTask.ExecuteAsync(IProgress`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.ScheduledTasks.ScheduledTaskWorker.ExecuteInternal(TaskOptions options)
[2024-05-13 17:56:36.731 +02:00] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Filesystem Scraper" Failed after 0 minute(s) and 0 seconds
```

**Comment by Cloud9Developer** [comment]
This should be resolved with the latest version (0.6.2.0).

**Comment by BLACK4585** [comment]
Yes it is, thank you!
